benchmarks/benchmark_registration.py

```diff
@@ -1,6 +1,11 @@
-from skimage.data import stereo_motorcycle
 from skimage.color import rgb2gray
-from skimage import registration
+
+try:
+    from skimage import registration
+    from skimage.data import stereo_motorcycle
```

Comment:
Should this import be moved outside of the try/except?

Reply:
I should just have put `from skimage import data` at the top and then called via `data.stereo_motorcycle` in the test cases. That is the pattern I followed in other files. It will raise `AttributeError` and fail the test case on older releases where this dataset is unavailable, but will not crash the whole test suite as occurs when an import failure occurs at the top.